.gitignore

```diff
@@ -209,6 +209,7 @@ tags
 /test/library/packages/HDF5/ex_lite1.h5
 /test/library/packages/HDF5/ex_lite3.h5
 /test/library/packages/ZMQ/getLastEndpointHelper
+/test/library/standard/FileSystem/chmod/chmod_noprivilege.execopts
```

Comment:
While we're looking at the .gitignore: you'll notice on line 219-220 of this file that `file` and `dir` from my original directory were listed in this file.  You'll want to remove those lists and add an equivalent one for this new directory.  Sorry I didn't catch that the first time!